src/flowr/internal-session.ts

```diff
@@ -84,7 +69,19 @@ export class FlowrInternalSession {
 			return a.location.start.line - b.location.start.line || a.location.start.column - b.location.start.column
 		})
 
-		const diagnostics: vscode.Diagnostic[] = []
+		this.outputChannel.appendLine('slice: ' + JSON.stringify([...result.slice.result]))
```

Comment:
we should only do diagonstics like this when desired - we may want to add a configuration which allows this.